src/sorespo/rfs.act

```diff
@@ -220,6 +341,11 @@ class BBInterface(base.BBInterface):
             # Create the interface
             unit = dev.configuration.interfaces.interface.create(i.name).unit.create("0")
             unit.description = "Link to {i.remote.device} [{i.remote.interface}]"
+            if i.monitor_health:
+                if dynstate is not None:
```

Comment:
Oh, is dynstate optional?

@ddjoh is the dynstate input always optional? I figure it should follow the YANG model, so the container itself is unconditionally there etc and values should reflect defaults etc

I reckon TTT should create the initial dynstate tree that's used before `update_dynstate()` has been called. WDYT?